homeassistant/components/suez_water/coordinator.py

```diff
@@ -48,61 +29,29 @@ def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry) -> None:
         )
 
     async def _async_setup(self) -> None:
-        self._sync_client = await self.hass.async_add_executor_job(self._get_client)
+        self._suez_client = await self._get_client()
```

Comment:
This can be inlined now. Also, make sure to raise `UpdateFailed` instead of `ConfigEntryNotReady` (I probably forgot to mention that the last time)

Reply:
just kicked the try/ catch as it's already handled in the lib 😄